package.json

```diff
@@ -36,8 +37,14 @@
   "types": "dist/index.d.ts",
   "exports": {
     ".": {
-      "require": "./dist/index.js",
-      "import": "./dist/index.mjs"
+      "browser": {
+        "require": "./dist/index-browser.js",
+        "import": "./dist/index-browser.mjs"
+      },
+      "default": {
+        "require": "./dist/index.js",
+        "import": "./dist/index.mjs"
+      }
```

Comment:
Is this something we should adopt for all of our libraries, e.g. in https://github.com/inrupt/solid-client-js/blob/cb19e7a13af5a609200775859e1f62693f79110a/package.json#L41?

Reply:
It's only needed when we want to have different entrypoints in node and browser.

In solid-client-js we want node and browser builds using the same entry point so it is not needed over there.